src/ProductCard.spec.js

```diff
@@ -1,14 +1,32 @@
 import { render, screen } from '@testing-library/react'
 import ProductCard from './ProductCard'
 
-describe('it shows ProductCard', () => {
+describe('render ProductCard', () => {
   const price = '18,-'
 
-  it('renders', () => {
+  it('shows ProductCard Images', () => {
     render(<ProductCard />)
 
-    const Image = screen.getByRole('img')
-    expect(Image).toBeInTheDocument()
+    const Images = screen.getAllByRole('img')
+    expect(Images).toHaveLength(2)
+  })
+
+  it('has the correct alt text', () => {
+    const altText = 'Lakritzel bottle'
+
+    render(<ProductCard path="https://localhost:3000/" alt={altText} />)
```

Comment:
Your `ProductCard` does not receive the props `path` and `alt`. So these two lines do almost nothing but render your `ProductCard` with no image. The correct way would be to add the path to the `image` prop. The `alt` attribute is hardcoded and you can't pass it as prop.
```suggestion
    render(<ProductCard image="https://localhost:3000/" />)
```